src/app-layout/visual-refresh/layout.tsx

```diff
@@ -63,37 +56,6 @@ export default function Layout({ children }: LayoutProps) {
     toolsHide
   );
 
-  /**
-   * Determine the offsetBottom value based on the presence of a footer element and
-   * the SplitPanel component. Ignore the SplitPanel if it is not in the bottom
-   * position. Use the size property if it is open and the header height if it is closed.
-   */
-  useLayoutEffect(
-    function handleOffsetBottom() {
-      let offsetBottom = footerHeight;
-
-      if (splitPanel && splitPanelPosition === 'bottom') {
-        if (isSplitPanelOpen) {
-          offsetBottom += splitPanelSize;
-        } else {
-          const splitPanelHeader = getSplitPanelHeader();
-          offsetBottom += splitPanelHeader ? splitPanelHeader.clientHeight : 0;
-        }
-      }
-
-      setOffsetBottom(offsetBottom);
-    },
-    [
-      footerHeight,
-      getSplitPanelHeader,
-      isSplitPanelOpen,
-      setOffsetBottom,
-      splitPanelPosition,
-      splitPanel,
-      splitPanelSize,
-    ]
```

Comment:
This effect exclusively depends on values coming from app layout context. Why calculating it here with an effect, if the same value can be pre-computed on `context.tsx` file already?